configuration/white_labels/tx.py

```
@@ -54,87 +54,44 @@ def get_white_label(self) -> WhiteLabel:
         },
-        "childSupport": {
-            "_label": "incomeOptions.childSupport",
-            "_default_message": "Child Support (Received)",
+        "investment": {
+            **ConfigurationData.income_options_by_category["investment"],
+            "pension": {
+                "_label": "incomeOptions.pension.tx",
+                "_default_message": "Military, Government, or Private Pension",
+            },
         },
+    }
```

Ruff (0.14.8)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)